pages/blog/_posts/2020-01-28-Thanos-infinite-metrics.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blogpost
+permalink: /blog/metrics-unlimited-thanos/
+title: "Towards Observability nirvana: Infinite metric retention with Thanos"
+date: 2020-01-28
+tags: infrastructure observability prometheus thanos kubernetes
+author: <a href="https://www.linkedin.com/in/hihrig/">Holger Ihrig</a>
+---
+In nowadays DevOps world our industry is relying on the ability to observe and monitorize our infrastructure and 
+services quite a bit. Working Group Two is no exception at this and as we are operating in the TelCo space
+we wanted to know more about the usage patterns of our platform throughout days, months and even years.
+
+Internally we have been running Prometheus for a long time with a fairly limited retention of 30 days. This did not
+allow us to look back far enough to make the observations we wanted to.
+Luckily for us there already was a solution out there that would fill our needs and in addition to that make our
+monitoring stack more resilient. The solution is called [Thanos](https://thanos.io/).
+
+## Thanos
+Thanos was originally developed by a company called [Improbable](https://improbable.io/) to achieve long term storage
+for Prometheus. It evolved into a much more complicated component wildly improving the scalability of the
+Prometheus monitoring Stack.
+
+The basic functionality however is that Thanos will upload the metrics collected by Prometheus onto any service with a 
+S3-compatible API or any other Storage Target supported by the Prometheus Remote Write feature. For readability we
```

Comment:
storage target and remote write don't need caps

Reply:
Done